mPower2/mPower2/AppDelegate.swift

```diff
@@ -80,6 +80,12 @@ class AppDelegate: SBAAppDelegate, RSDTaskViewControllerDelegate {
         
         let retval = super.application(application, willFinishLaunchingWithOptions: launchOptions)
         
+        // The SBAAppDelegate does not refresh the app config if we already have it
```

Comment:
This is an issue I noticed when developing Psorcast, even if you bump app config versions, it doesn't update if there is one in the cache, so this just refreshes it from bridge if we have a cached version.